django/contrib/staticfiles/storage.py

```diff
@@ -172,6 +175,24 @@ def converter(matchobj):
             if url.startswith('/') and not url.startswith(settings.STATIC_URL):
                 return matched
 
+            # Ignore css statement, rule which is inside css comment
+            comment_regex = re.compile('/\\*[^*]*\\*+(?:[^/*][^*]*\\*+)*/')
```

Comment:
I think we should use a bit more complicated regular expression `\/\*[^*]*\*+([^/*][^*]*\*+)*\/` (see ["lexical scanner"](https://www.w3.org/TR/CSS22/grammar.html#scanner) and ["consume comments"](https://drafts.csswg.org/css-syntax-3/#consume-comment)). Also, we should take into account only comments that contain `url` or `import`.